packages/libf2c/meta.yaml

```diff
@@ -1,19 +1,16 @@
 package:
-  name: CLAPACK
-  version: 3.2.1
+  name: libf2c
```

Comment:
Why is this called libf2c? Also, can you explain somewhere why we need this in addition to openblas?

Reply:
We are only using the f2clib part of CLAPACK so I renamed these files, it would have been weird in my opinion to have CLAPACK and OpenBLAS.

The f2c part is needed for the f2ced Fortran files in scipy for example to define things like `c_abs`. Note that the f2clib.a symbols are added in libopenblas.so in the OpenBLAS meta.yaml.

Not sure where to add this kind of info, maybe in the meta.yaml? Let me know if you see a good place for this!

Reply:
> in the meta.yaml?

Yeah as a comment at the top of `meta.yaml` works.